tests/unit/scripts/test_initialize_library.py

```diff
@@ -9,10 +9,10 @@
 def test_init_library():
     # Create the user agains the current mongo database
     with patch('pymongo.MongoClient') as MongoClient, \
-         patch('arctic.scripts.arctic_init_library.logger', autospec=True) as logger, \
-         patch('arctic.scripts.arctic_init_library.Arctic', autospec=True) as Arctic, \
-         patch('arctic.scripts.arctic_init_library.get_mongodb_uri', autospec=True) as get_mongodb_uri, \
-         patch('arctic.scripts.arctic_init_library.do_db_auth', autospec=True) as do_db_auth:
+         patch('arctic.scripts.arctic_init_library.logger') as logger, \
```

Comment:
When Mock went to 1.1.0 they changed how autospecing works, which now affects methods wrapped in decorators. I'm putting these back to see if they work, if not it will just take some more massaging to get them working. Specin'g seems to work, but autospecing does not (when decorated methods come into play). see http://stackoverflow.com/questions/34800832/issue-mocking-with-a-decorated-method-call-in-python-2-7